cluster/manifests/skipper/deployment.yaml

```diff
@@ -133,6 +133,11 @@ spec:
             secretKeyRef:
               name: skipper-ingress
               key: datadome-api-key
+        - name: KASADA_API_KEY
+          valueFrom:
+            secretKeyRef:
+              name: skipper-ingress
+              key: kasada-api-key
```

Comment:
I assume this will only be relevant for some clusters for PoC, will skipper-ingress handle this correctly if the secret value is `""` (empty string?)

Reply:
Yes that's not a problem. This is only loaded in case of a lua filter that will be deployed via internal. The same way we did with datadome.